static/src/styles/core/interaction.styl

```diff
@@ -0,0 +1,13 @@
+.Interaction__bigButton {
```

Comment:
Great work abstracting these out into reusable components 😄 

_Tiny_ nit here. `bigButton` is technically a block, not an element (but is being named as an element in this case). It seems like you're using the block segment (`Interaction` in this case) as a namespace. I appreciate the need for namespaces, and maybe we do need them, so in this case, `BigButton` or `InteractionBigButton` would match our styleguide.